### DISCUSSION TITLE
No mouse... help!

### DISCUSSION THREAD

**Discussion by J534** [Stellarium/stellarium#3662] [General]
Just installed Stellarium on windows 11. When I open the program I have no visible mouse. But it does seem my mouse is "there" because I can mouse over menus. Anyone know how to fix?

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#mouse-cursor-invisible-windows

**Comment by J534** [comment]
**This is a known long-time defect for the default OpenGL mode on Windows when you have Mouse Trails enabled or another reason to use a "software cursor", like f.lux screen color temperature modifier. There are 2 workarounds:

1) Use ANGLE mode when you need mouse trails or f.lux. 2) Make sure to switch off mouse trails to avoid this problem.**

I'm confused. It doesnt look like I have mouse trails enabled on windows.

**Reply by gzotti** [reply]
There may be _'another reason to use a "software cursor"'_ then. Adhere to the core of suggestion (1) if you cannot follow along (2) which may read "switch off whatever it is that causes the issue on your system to avoid this problem". 
Note that ANGLE mode is available on Qt5-based builds only.

**Reply by J534** [reply]
What is ANGLE mode and how do I enable it? Also I'm a bit confused why I would install f.lux? It looks like its a monitor color temperature program.
